assets/src/components/nav/topNav.tsx

```diff
@@ -49,8 +50,24 @@ const TopNav = (): JSX.Element => {
               placement="bottom"
             >
               <Popover className="c-top-nav__popover">
-                <Popover.Body>
-                  <LoggedInAs email={email} />
+                <Popover.Body className="p-0">
+                  <Dropdown.Menu
+                    show={showUserPopover}
+                    className="position-static"
+                  >
+                    <Dropdown.ItemText className="w-auto">
+                      <LoggedInAs email={email} />
+                    </Dropdown.ItemText>
+                    <Dropdown.Divider />
+                    <Dropdown.Item
+                      href="/auth/keycloak/logout"
+                      className="w-auto"
+                    >
+                      <span className="d-flex align-items-center">
+                        <BsIcon.BoxArrowRight className="me-2" /> Log out
+                      </span>
```

Comment:
question/suggestion: what is this `<span/>` doing? Could this be replaced with `icon-link`?
```suggestion
                      className="w-auto icon-link"
                    >
                      <BsIcon.BoxArrowRight className="me-2" /> Log out
```

Reply:
Yes, good call. I'm not going to commit this suggestion directly because it turns out I had to tweak some other styling and utility classes as well to make everything work out (though those changes ended up fixing another little annoying style detail I was trying to debug).

Reply:
1dc9141b